eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -99,6 +100,25 @@ stages:
         overrideDistroDisablingSha1: false # 🚫
         runOnline: true                    # ✅
         withPreviousSDK: true              # ✅
+        useMonoRuntime: false              # 🚫
+
+    - template: ../jobs/vmr-build.yml
```

Comment:
Is the intent to run this on all PRs or is this running in PRs just to validate?

Reply:
Excuse me, I see this is outside of PR validation.